cpp/examples/sbml2memilio.h

```diff
@@ -0,0 +1,856 @@
+#include <cmath>
```

Comment:
Also, is this in not an example in my opinion. It just more a file to transform sbml models into memilio models. Dont think this should be part of the example dir. Maybe we should discuss where to place it.